flytekit/types/flyte_file.py

```diff
@@ -15,6 +16,20 @@ def noop():
     ...
 
 
+def unarchive_file(local_path: str, to_dir: str):
```

Comment:
don't feel like this belongs in this file.

Reply:
it does not, but we do not have ioutils at the moment